### ISSUE TITLE
can you help me with my script?

### ISSUE THREAD

**Issue by daisuke031** [Masuzu/ZooeyBot#823]
hi there, im new to this scripting deal, but I've been in love with since I started. 
that said, which statement would I write for the bot check if my characters have a status ailment and do an action/don't do any actions? I looked through the examples on here but couldn't really find the propper statement...?
this is what I tried at first but I think it's completly useless:

> >

if characters["Cagliostro"] ~= nil 
 if not characters["Cagliostro"]:HasStatusEffect("1263") then
 characters["Cagliostro"]:WithWaitTime(3200):UseSkill(2)
 end
end
>
that's part of the script I was using for this GW to work agaisnt the sleep status. I thought the ``if not`` statement would check if the character DOESNT have the status and then do the action, but it didn't work, since she tried to use the skill, which she wasn't able to, the skill pop up showed up and everything went downhill.
I also tried the ``avaliability`` statement before, but same result. I'm almost certain this is a rookie mistake, but... pls help.

**Comment by ErogeSeiyuu** [comment]
I would write it as 

```
if characters["Cagliostro"] ~= nil and characters["Cagliostro"]:HasStatusEffect("1263") == false then
         characters["Cagliostro"]:WithWaitTime(3200):UseSkill(2)
end
```

Can you test if HasStatusEffect == false work?

**Comment by daisuke031** [comment]
so I tried testing it on a more direct way using the Phoebe character to se if she would pop her third skill upon recognizing the team were under sleep, and it also didnt work. I also tried using `not HasStatus...` same results. I wonder if the sleep ID it's correct, but I don't think I have the `ListenerMode`.
![image](https://user-images.githubusercontent.com/39315856/40638725-49c87282-62e3-11e8-918b-7b4b1b5dc5cb.png)
`character_3:WithWaitTime(3200):UseSkill(3)`
 ` character_3:WithWaitTime(3200):UseSkill(1)`
` if characters["Owen"] ~= nil and characters["Owen"]:HasStatusEffect("1263") == true then`
` character_3:WithWaitTime(3200):UseSkill(2)`
` end`
